### PR TITLE
Add template for bird-clef challenge

### PR DESCRIPTION
**Pull request by anlthms**

### CODE REVIEW

**Comment by anlthms** [comment]
no need to train to completion. maybe train for a few epochs and make a submission to verify that the pipeline works.

**Comment by hornof** [comment]
When I choose Code Templates, it looks like it gets stuck on "Loading Templates", and it doesn't finish rendering. I'm seeing this on both prod and dev.

image

**Comment by hornof** [comment]
Seeing this in the Javascript console.  Will continue to debug.

image

File: contents.json [hunk 1]
```diff
@@ -24,19 +26,31 @@
...
             "directory": "pytorch-kaggle-sorghum",
             "tags": [ "PyTorch", "timm", "Computer Vision", "Classification", "Kaggle" ]
-        }
+        },
+        {
+            "name": "BirdCLEF 2022",
+            "description": "Identify bird calls in soundscapes",
+            "directory": "pytorch-kaggle-bird-clef",
+            "tags": [ "PyTorch", "timm", "Audio", "Classification", "Multi-label"]
+        },
```
Comment (rgharris) [inline contents.json:54]: JSON does not allow trailing commas, that is leading to the issues @hornof is running into.

**Review by rgharris — COMMENTED** [review]
There is an issue with `contents.json`.

**Comment by hornof** [comment]
README.md suggestions:

> - On the `Import Data` tab, choose Kaggle and then enter `birdclef-2022`.
- On the `Import Data` tab, choose Kaggle Competition Data and then enter `birdclef-2022`.

> - Run [install.sh](install.sh) as a custom experiment to install required packages on the Compute Server.
- Install required packages on the Compute Server using the `Run Experiment` menu and select `Custom > install.sh`

> - Train a model using the `Run Experiment` menu.
- Train a model using the `Run Experiment` menu and select`Train > Start Full Training`.

> - Run [kaggle.sh](kaggle.sh) as a custom experiment to upload the code to Kaggle as a dataset.
-  Upload the code to Kaggle as a dataset by using the `Run Experiment` menu and select `Custom > kaggle.sh`

Again, I got a blank screen.  After waiting for a few minutes I hit refresh and the console correctly displayed the "Press enter to upload ...  " and everything worked fine after that.

> - To create a submission, copy [kaggle.ipynb](kaggle.ipynb) to a new Kaggle notebook.

Good.

> - Add the notebook output of `https://www.kaggle.com/luminide/wheels1` as Data.

Good.

> - Add your dataset at `https://www.kaggle.com/<kaggle_username>/kagglecode` as Data.

Good.

> - Add the relevant competition dataset as Data.
- Add the `birdclef-2022` competition dataset as Data.

> - Run the notebook after turning off the `Internet` setting.

Good.

> - Check the [leaderboard](https://www.kaggle.com/competitions/birdclef-2022/leaderboard) to see your score!

Do I just wait... or am I do I need to submit manually, e.g.

image

**Comment by anlthms** [comment]
made those changes and pushed a commit.
yes, you do have to submit (and then wait for 2 hours!)

**Comment by hornof** [comment]
Yeah, after a while I did that... and it's been spinning away ever since.  Still waiting... 

So maybe update this comment.  Something like:

> - Run the notebook after turning off the Internet setting.

Run the notebook after turning off the Internet setting and submit the result (note -- this may take a couple hours).

**Review by hornof — APPROVED** [review]

**Comment by anlthms** [comment]
- set up to train on 100% of the data
- reduced fast sweep epochs to 10

**Pull request merged**